CHANGELOG.md

```diff
@@ -1,66 +1,60 @@
-# TiDB Change Log
-All notable changes to this project will be documented in this file.
-See also [Release Notes](https://github.com/pingcap/docs/blob/master/releases/rn.md)
-
-[tikv_change_log]: https://github.com/pingcap/tikv/CHANGELOG.md
-[pd_change_log]: https://github.com/pingcap/pd/CHANGELOG.md
+# TiDB Changelog
+All notable changes to this project will be documented in this file. See also [Release Notes](https://github.com/pingcap/docs/blob/master/releases/rn.md), [TiKV Changelog](https://github.com/pingcap/tikv/blob/master/CHANGELOG.md) and [PD Changelog](https://github.com/pingcap/pd/blob/master/CHANGELOG.md).
 
 ## [2.0.0-rc.5] - 2018-04-17
 ### New Features
-* Support showing memory usage of the executing statements in the Show Process List statement
-* Support setting the table comment using the Alter statement
-### Improves
-* Clean up the written data while rolling back the Add Index operation, to reduce consumed space
+* Support showing memory usage of the executing statements in the `Show Process List` statement
```

Comment:
I think if we add `` for the statement, this should be write like `SHOW PROCESSLIST` or `show processlist`.